helm-chart/values.yaml

```diff
@@ -222,6 +222,57 @@ cvat:
           protocol: TCP
           name: http
 
+  kvrocks:
+    enabled: true
+    external:
+      host: 127.0.0.1
+    existingSecret: "cvat-kvrocks-secret"
+    secret:
+      create: true
+      name: cvat-kvrocks-secret
+      password: cvat_kvrocks
+    replicas: 1
```

Comment:
Does it actually work if you set this parameter to more than one? I would assume running Kvrocks with replication is more involved than that. And if it doesn't work, then we shouldn't provide the option.